plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -35,6 +35,29 @@ void testShowTables()
         assertUpdate("CREATE TABLE faker.default.test (id INTEGER, name VARCHAR)");
         assertTableColumnNames("faker.default.test", "id", "name");
     }
+    
+    @Test
+    void testRenameTable()
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
+        assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
+        assertQuery("SHOW TABLES FROM faker.default", "VALUES 'original_table'");
+        assertUpdate("ALTER TABLE faker.default.original_table RENAME TO renamed_table");
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'renamed_table'", "VALUES 'renamed_table'");
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
+        assertUpdate("DROP TABLE faker.default.renamed_table");
+    }
+    
+    @Test
+    void testAlterTableSetProperties() 
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'test_table'", "SELECT '' WHERE false");
+        assertUpdate("CREATE TABLE faker.default.test_table (id INTEGER, name VARCHAR)");
+        assertUpdate("ALTER TABLE faker.default.test_table SET PROPERTIES default_limit = 100");
+        assertQuery("SELECT table_property_name, table_property_value FROM information_schema.tables WHERE table_name = 'test_table' AND table_property_name = 'default_limit'",
```

Comment:
The table_property_name column doesn't exist. I think you have to execute a show create table query instead